### PR TITLE
add gaia_defaults.json on install

### PR DESCRIPTION
**Pull request by nstarman**

### CODE REVIEW

**Comment by jobovy** [comment]
@webbjj, could you check that this works?

**Comment by webbjj** [comment]
I receive the same error as before. The notebook make_gaia_query_examples.ipynb
works for me when run in gaia_tools/examples. But it does not work in a
separate directory.

**Comment by nstarman** [comment]
maybe since there isn't an __init__.py in the folder defaults/ it doesn't install correctly. I'll push a fix.

**Comment by webbjj** [comment]
To clarify, I cloned and installed Nathanials master. Is this correcT?

Jeremy

File: setup.py [hunk 1]
```diff
@@ -13,7 +13,8 @@
       package_dir = {'gaia_tools/': ''},
       packages=['gaia_tools','gaia_tools/load','gaia_tools/query',
                 'gaia_tools/xmatch','gaia_tools/select','gaia_tools/util'],
-      package_data={'gaia_tools/select':['2massc_hp5.txt'],
+      package_data={'gaia_tools/select': ['2massc_hp5.txt'],
+                    "gaia_tools/query/defaults": ["gaia_defaults.json"],
```
Comment (jobovy) [inline setup.py]: Shouldn't this be
```
"gaia_tools/query": ["defaults/gaia_defaults.json"]
```
?

Comment (nstarman) [reply setup.py]: done. And made the defaults directory platform agnostic.

Comment (jobovy) [reply setup.py]: Thanks! Could you remove the ``defaults/__init__.py`` file again?

**Comment by jobovy** [comment]
@webbjj, please try again.

**Comment by webbjj** [comment]
Apologies for late reply, it seems to work now. Thanks for dealing with it
so quickly.

Jeremy

**Comment by jobovy** [comment]
Thanks, have merged!

**Pull request merged**